src/server.c

```diff
@@ -625,6 +625,17 @@ const char *strChildType(int type) {
     /* clang-format on */
 }
 
+
+const char *strRDBLoadType(int type) {
+    /* clang-format off */
+    switch(type) {
+    case RDB_LOAD_TYPE_PARSER: return "parser";
+    case RDB_LOAD_TYPE_DISK: return "disk";
+    default: return "unknown";
```

Comment:
I wonder if we should not just assert on any other value? this seems like a bug reaching the point of non defined target